dotcom-rendering/src/server/cmp-dummy.ts

```diff
@@ -0,0 +1,16 @@
+import type { CMP } from '@guardian/consent-management-platform/dist/types';
+
+export const cmp: CMP = {
+	__disable: () => {},
+	__enable: () => {},
+	__isDisabled: () => false,
+	hasInitialised: () => false,
+	init: () => {},
+	showPrivacyManager: () => {},
+	version: 'n/a',
+	willShowPrivacyMessage: () => Promise.resolve(false),
+	willShowPrivacyMessageSync: () => false,
+};
+
+export const onConsentChange = () => {};
+export const getConsentFor = () => {};
```

Comment:
Would it be worth making it so if any of these functions are called, they warn the user about (or throw an error regarding) not using CMP on the server?

You could also do get() methods for properties too but this might be overkill

Reply:
Excellent idea. I was wondering if we liked this approach before committing more time to it.